### PR TITLE
Eclipse static code anlysis warnings fixes

### LINKED ISSUE #162: Warnings in Eclipse
Eclipse displays several warnings on the project:

 - Resource leak: 'stdErr' is never closed	DefaultProcessRunner.java	/appengine-plugins-core/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process	line 133

- Resource leak: 'stdOut' is never closed	DefaultProcessRunner.java	/appengine-plugins-core/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process	line 117

- The import com.google.cloud.tools.appengine.cloudsdk.CloudSdk is never used	DefaultProcessExitListener.java	/appengine-plugins-core/src/main/java/com/google/cloud/tools/appengine/cloudsdk/process	line 19

- The import com.google.common.collect.ImmutableSet is never used	DevAppServerArgs.java	/appengine-plugins-core/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args	line 19

- The import java.util.Set is never used	DevAppServerArgs.java	/appengine-plugins-core/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args	line 22

- The value of the local variable exitCode is not used	DefaultProcessRunner.java	/appengine-plugins-core/src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process	line 165

- There is no schema defined for this pom.xml.	pom.xml	/appengine-plugins-core	line 1

### PR DESCRIPTION
**Pull request by meltsufin**
Fixes #162 

@akerekes please verify that this actually resolves all of the warnings

* Resource leak: 'stdErr' is never closed DefaultProcessRunner.java

* Resource leak: 'stdOut' is never closed DefaultProcessRunner.java

* The import com.google.cloud.tools.appengine.cloudsdk.CloudSdk is never
used DefaultProcessExitListener.java

* The import com.google.common.collect.ImmutableSet is never used
DevAppServerArgs.java

* The import java.util.Set is never used DevAppServerArgs.java

* The value of the local variable exitCode is not used
DefaultProcessRunner.java

* There is no schema defined for this pom.xml. pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,4 +1,8 @@
-<project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (elharo) [inline pom.xml]: we've been back and forth on whether we want/need these attributes. Not all eclipse suggestions are necessarily good ideas.

Comment (meltsufin) [inline pom.xml]: Do you have a ref to the previous discussions? I would be curious to see what the pros and cons of including the namespace and schema are for pom.xml?

Comment (elharo) [inline pom.xml]: The xmlns attribute is defintiely a good idea, IMHO. The other two are at best optional.

**Comment by akerekes** [comment]
There's an unused import here: https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/blob/d14ce993d5770e414c856c2bcc4b075e18d3d9d2/src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineStandardStaging.java#L29

otherwise LGTM

**Comment by elharo** [comment]
LGTM

**Pull request merged**